package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@cosmjs/cosmwasm-stargate": "^0.32.2",
+    "@iarna/toml": "^2.2.5",
```

Comment:
this doesn't seem to be maintained anymore, how about this [package](https://www.npmjs.com/package/smol-toml) instead? not a huge issue since it's just toml parsing

Reply:
will try